guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -14,18 +14,20 @@ include::snip_firewalld.adoc[]
 [options="nowrap"]
 ----
 # firewall-cmd \
---add-port="53/udp" --add-port="53/tcp" \
---add-port="67/udp" \
---add-port="69/udp" \
---add-port="80/tcp" --add-port="443/tcp" \
+--add-service=dns \
+--add-service=dhcp \
+--add-service=tftp \
+--add-service=http \
+--add-service=https \
 ifdef::katello,satellite,orcharhino[]
 --add-port="5647/tcp" \
```

Comment:
Same here: katello-agent is not enabled by default so qpid is not there. Should this be open by default, or should it be part of the procedure to enable katello-agent?

Reply:
I would like that as well, but I don't have time to work on it myself now and prefer to at least get this in. Then open an issue for the follow up work.